.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: dart
 sudo: required
 dart:
-  - 2.4.1 # TODO: Remove this and bump the lower SDK bound to 2.6.1 once we've upgraded the ecosystem to it
   - stable
```

Comment:
@evanweible-wf has been moving some of our OSS repos to github actions as he's been updating them to NNBD. Should we do the same with this repo?

Reply:
If it's ok, I think I'd like the CI changes to be in a separate PR.